src/GuppyDataExplorer/ExplorerButtonGroup/index.jsx

```diff
@@ -344,9 +344,13 @@ class ExplorerButtonGroup extends React.Component {
   };
 
   downloadData = (filename, fileFormat) => () => {
-    const fileTypeKey = fileFormat.toUpperCase();
-    const isJsonFormat = fileTypeKey === 'JSON' || fileTypeKey === 'DATA';
-    this.props.downloadRawData({ format: fileTypeKey }).then((res) => {
+    const fileTypeKey = fileFormat.toLowerCase();
+    const isJsonFormat = fileTypeKey === 'json' || fileTypeKey === 'data';
+    const queryArgObj = {};
+    if (fileTypeKey !== 'data') {
+      queryArgObj.format = fileTypeKey;
+    }
+    this.props.downloadRawData(queryArgObj).then((res) => {
       if (res) {
         const blob = new Blob([isJsonFormat ? JSON.stringify(res, null, 2) : res], { type: `text/${isJsonFormat ? 'json' : fileFormat.toLowerCase()}` });
```

Comment:
Since you are using the `toLowerCase()` for the `fileTypeKey` you could just use it for the blob too instead of calling twice.

Reply:
what do you mean by "calling twice"?